### DISCUSSION TITLE
How to resolve this?

### DISCUSSION THREAD

**Discussion by prm888** [lllyasviel/Fooocus#3711] [General]
![image](https://github.com/user-attachments/assets/ff7d1546-f092-4c9e-a036-0171f3d0fa83)

Please see above. I tried using fooocus colab but encountering disconnections after atleast 5 tries then suddenly the above appears. How to resolve?

**Comment by NemesisPrasa** [comment]
got the same error! Donno how to fixed this

**Reply by Rogala** [reply]
This will not be fixed, the developer is no longer interested in this product

**Reply by mashb1t** [reply]
@Rogala Fooocus is considered to be a complete product, which does only receive critical bugfixes if any arise. Upgrading Gradio is a major task, which has already been started but ultimately stopped again due to the needed investment of time being too high in comparison to the result for the community.

=> Fooocus will continue using 3.41.2 and not 4.x

**Reply by Rogala** [reply]
@mashb1t I know the bulk of it is complete, but there were plans for optimization and tweaking. You're coming back?

**Reply by mashb1t** [reply]
@Rogala I've handed over the code ownership to lllyasviel again and fooocus is in LTS bugfix-only support => there were and are no plans for tweaking and fiddling i am aware of.

**Comment by ghost** [comment]
Fooocus-colab is a separate unofficial project.
